src/index.html

```diff
@@ -2,16 +2,167 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-box">
+        <legend>
+          Personal information
+        </legend>
+        <label class="form-field">
+          Surname:
+          <input 
+          type="text"
+          name="Surname" 
+          autocomplete="off" 
+          required
+          >
```

Comment:
```suggestion
          <input 
            type="text"
            name="Surname" 
            autocomplete="off" 
            required
          >
```

![image](https://user-images.githubusercontent.com/80700247/222440280-056a170a-a805-480f-8ac8-ecf279edaf5c.png)

Fix everywhere